workspace/apps/pidp/src/app/features/auth/pages/login/login.page.ts

```diff
@@ -40,12 +51,31 @@ export class LoginPage {
     this.specialAuthoritySupportEmail =
       this.config.emails.specialAuthoritySupport;
     this.idpHint = routeSnapshot.data.idpHint;
+    this.collectionNotice = documentService.getAuthCollectionNotice();
   }
 
   public onLogin(): void {
-    this.authService.login({
-      idpHint: this.route.snapshot.data.idpHint,
-      redirectUri: this.config.applicationUrl,
-    });
+    const data: DialogOptions = {
+      title: 'Collection Notice',
+      component: HtmlComponent,
+      data: {
+        content: this.collectionNotice,
+      },
+    };
+
+    this.busy = this.dialog
```

Comment:
Both users and admins login using this method and the admins don't need to see the collection notice